### DISCUSSION TITLE
My application has been rejected more than 5 times!

### DISCUSSION THREAD

**Discussion by KrackF1** [community/community#53684] [GitHub Education]
My application was rejected for a reason - you are unlikely to be verified until you fill out your GitHub user profile with your full name, as indicated in your academic affiliation document, plus a short biography.
What do I need to change so that my application is accepted?

**Comment by samus-aran** [comment]
Hi @Krack777 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 

Hope this helps! Let us know how you get along 👍
